packages/block-library/src/image/save.js

```diff
@@ -65,6 +66,14 @@ export default function save( { attributes } ) {
 		</>
 	);
 
+	if ( isListItem ) {
+		return (
+			<li { ...useBlockProps.save() }>
```

Comment:
Nice catch ... do you have any thoughts on an alternative approach?

Reply:
> Nice catch ... do you have any thoughts on an alternative approach?

Can we just move the `useBlockProps.save()` above the conditional, storing it in a variable, then destructure it as required?

Reply:
thats what I was thinking too  - @ZebulanStanphill can you see any issues with just doing that?

Reply:
There's no problems with that solution. :+1:

Actually, I just realized that `useBlockProps.save` isn't even an actual React hook. It's actually an alias for [`getBlockProps`](https://github.com/WordPress/gutenberg/blob/0ca97366d0e6c2020dde9015bace9d9c1c970a1d/packages/blocks/src/api/serializer.js#L78). So I think there's actually nothing technically wrong here. Still, I think it's possible that it might turn into an actual hook in the future, and it's kind of confusing to begin with, so I'd go ahead and move it out of the conditional anyway.

Reply:
Fixed.